dev/dags/.airflowignore.old

```diff
@@ -0,0 +1,12 @@
+basic_cosmos_dag.py
+#basic_cosmos_task_group.py
```

Comment:
Should we uncomment this?

Reply:
I remove the file, so this no longer exists :)